### PR TITLE
[CC-1377] add new paypage

### PR DESCRIPTION
**Pull request by Ryouzanpaku**

### CODE REVIEW

File: test/integration/Resources/PaypageV2Test.php [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+    /** @test
+     * @group skip
+     */
+    public function createPaypageWithRiskData()
+    {
+        $unzer = $this->getUnzerObject();
+        $risk = new RiskData();
+
+        $risk->setThreatMetrixId('f544if49wo4f74ef1x')
```
Comment (AntonAtUnzer) [inline test/integration/Resources/PaypageV2Test.php:221]: We only support these fields in risk:
 "risk": {
    "registrationLevel": "string",
    "registrationDate": "string",
    "customerGroup": "top",
    "confirmedOrders": 0,
    "confirmedAmount": 0
  }

customerId and threatmetrixId are gone.
ThreadMetrixId will be generated by the Paypage UI

File: src/Resources/EmbeddedResources/Paypage/PaymentMethodConfig.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+use UnzerSDK\Resources\AbstractUnzerResource;
+
+class PaymentMethodConfig extends AbstractUnzerResource
+{
+    protected ?bool $enabled = null;
+    protected ?int $order = null;
+
+    // type specific configs.
+    protected ?string $label = null; // paylater types only.
+    protected ?string $checkoutType = null; // paypal only.
```
Comment (AntonAtUnzer) [inline src/Resources/EmbeddedResources/Paypage/PaymentMethodConfig.php:14]: doesn't exist for paypal. Express doesn't make sense for PP

Comment (AntonAtUnzer) [reply src/Resources/EmbeddedResources/Paypage/PaymentMethodConfig.php:14]: However, a checkoutType on the create paypage call exists full/payment_only/no_shipping

File: src/Resources/V2/Paypage.php [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+class Paypage extends AbstractUnzerResource
+{
+    public const URI = '/merchant/paypage';
+
+    /** @var string $mode "charge" or "authorize" */
+    protected string $mode;
+    protected float $amount;
+    protected string $currency;
+
+    protected ?string $type = null;
```
Comment (AntonAtUnzer) [inline src/Resources/V2/Paypage.php:26]: checkoutType as well e.g. full, payment_only, no_shipping

File: test/integration/Resources/PaypageV2Test.php (hunk 1 above)
Comment (Ryouzanpaku) [reply test/integration/Resources/PaypageV2Test.php:221]: @AntonAtUnzer
I see, I reused the risk object for normal transactions here. So i can not remove them from the original class.
The merchant could just not set the properties. Or should i make sure that he is not setting them for paypage requests?

File: src/Resources/EmbeddedResources/Paypage/PaymentMethodConfig.php (hunk 2 above)
Comment (Ryouzanpaku) [reply src/Resources/EmbeddedResources/Paypage/PaymentMethodConfig.php:14]: Ok, I can remove it.
that means the checkouttType for methodconfig will also be removed on paypage service?
I saw it just for paypal there.

File: src/Resources/V2/Paypage.php (hunk 3 above)
Comment (Ryouzanpaku) [reply src/Resources/V2/Paypage.php]: - Add checkout type to paypage.
- Remove checkout type from method config.

**Review by AntonAtUnzer — APPROVED** [review]

**Pull request merged**